index.html

```diff
@@ -1,16 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Watch&Chill App</title>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
+    <link rel="stylesheet" href="./assets/css/style.css">
+
+
+
+    <title>Watch&Chill App</title>
+
+
 </head>
+
 <body>
     <nav></nav>
-    
+    <div>
+        <h1 class="title has-text-centered">Welcome to Watch and Chill!
+        </h1>
+
+        <div class="columns is-8">
+            <div class="column is-half is-offset-one-quarter">
+                <p class=" is-size-3 has-text-centered">
+                    Watch & Chill was made for people that loves movies but do not know what to watch, so we are here to
+                    help
+                    you,
+                    you just
+                    need to select your favorite genres and will do the rest. Enjoy ;)!
+                </p>
+            </div>
+        </div>
+        <!-- instead of a picture i belive we should put the Carousel to give a better image to the page. -->
+        <div class="column is-one-quarter">
+            <div class="card equal-height">
+                <div class="card-image has-text-centered">
+                    <picture>
+                        <img class="is-centered"
+                            src="https://media.thetab.com/blogs.dir/117/files/2016/03/hqdefault.jpg"
+                            alt="watch and chill image">
+                    </picture>
+
+                </div>
+
+            </div>
+
+        </div>
+        <div class="has-text-centered">
+            <a href="./genres.html" class="button is-success is-large">
```

Comment:
Excellent approach, although two things are going to be needed here since I merged some changes in main to avoid conflicts:

- As you are reaching the next page by a link <a> button the js file and folder is not longer needed, would be better to remove the folder and file that you created (js/home.js), unless you have a purpose for that file it is ok to leave it.
- With the merged code from main genres page is not longer in root folder, please update the path to reach genre page, right now if you click in "Get Started" button is taking you to a page that does not exist.

Reply:
Files have been the folder and file (js/home.js) have been deleted. 

also now the get started button do takes you to genres page, instead of an empty page.